packages/core-ui/styles/buttons.scss

```diff
@@ -296,4 +296,15 @@ SPDX-License-Identifier: Apache-2.0 */
   &.active {
     background: var(--button-secondary-background);
   }
-}
+  div {
```

Comment:
This style will take place for all `div`s inside this class. Can you give it a unique class name or use `>` to assign a particular child.